tests/Unit/Ads/AccountServiceTest.php

```diff
@@ -472,6 +478,66 @@ public function test_get_billing_status_pending() {
 		);
 	}
 
+	public function test_get_ads_account_has_access() {
```

Comment:
By the name of this function I'm not able to tell what this test is supposed to return. Based on the contents it should be a test for `does_not_have_access`. If so then we are still missing the for a scenario where `has_access` returns true.

Reply:
I've added a new test for when `has_access` returns true as part of the changes to include the `step` in the response (see: [cfbdb3b](https://github.com/woocommerce/google-listings-and-ads/pull/2270/commits/cfbdb3b139952e9a866e837164bc3fb8ea4ff605)). I've renamed all of these associated tests to start with the method name they are testing (i.e. `test_get_ads_account_has_access`) with the scenario being tested added to the end, for all tests that do not return true for `has_access`.